server/routes/index.ts

```diff
@@ -227,6 +228,7 @@ export default function routes(services: Services): Router {
   router.use(professionalContactsRouter(services))
   router.use(goalsRouter(services))
   router.use(locationDetailsRouter(services))
+  router.use(probationDocumentsRouter(services))
```

Comment:
I think David mentioned that routes with only one related endpoint as we have here should be inline in this file, but I do like the idea of organising in separate files, even in this case - maybe worth another quick chat in our catch up on Monday?